fusionauth/resource_fusionauth_api_key.go

```diff
@@ -98,11 +103,19 @@ func resourceAPIKey() *schema.Resource {
 								"/api/passwordless/start",
 								"/api/prometheus/metrics",
 								"/api/reactor",
+								"/api/reactor/metrics",
 								"/api/report/daily-active-user",
 								"/api/report/login",
 								"/api/report/monthly-active-user",
 								"/api/report/registration",
 								"/api/report/totals",
+								"/api/scim/resource/v2/EnterpriseUsers",
+								"/api/scim/resource/v2/Groups",
+								"/api/scim/resource/v2/Me",
+								"/api/scim/resource/v2/ResourceTypes",
+								"/api/scim/resource/v2/Schemas",
+								"/api/scim/resource/v2/ServiceProviderConfig",
+								"/api/scim/resource/v2/Users",
```

Comment:
These don't use a normal API key, they should be removed from this list. 

It  looks like this is a bug in our admin UI as well. We'll remove them on our end as well.

Reply:
Ah gotcha. I have updated now!